pkg/v1/providers/Makefile

```
@@ -28,7 +28,11 @@ IMAGE_REPO ?= vmware.io
 PROVIDER_TEMPLATE_IMG_TAG ?= latest
 
 
-FILES_TO_IGNORE='.env\|.idea\|.git\|.github\|hack\|.yamllint\|.gitlab\|Makefile\|provider-bundle\|tests\|README.md\|client\|go.mod\|go.sum\|providers.sha256sum|\Dockerfile.templates'
+FILES_TO_IGNORE='.env\|.idea\|.git\|.github\|hack\|.yamllint\|.gitlab\|Makefile\|provider-bundle\|tests\|README.md\|client\|go.mod\|go.sum\|providers.sha256sum\|Dockerfile.templates'
+ifeq ($(EXCLUDE_CLUSTER_CLASS_FILES), true)
+FILES_TO_IGNORE='.env\|.idea\|.git\|.github\|hack\|.yamllint\|.gitlab\|Makefile\|provider-bundle\|tests\|README.md\|client\|go.mod\|go.sum\|providers.sha256sum\|yttcc\|yttcb\|cconly\|cc.yaml\|Dockerfile.templates'
+endif
```

I think there's a more "GNU Make" to do this. Let me experiment and come back in a minute.

---

```suggestion
FILES_TO_IGNORE?=.env .idea .git .github hack .yamllint .gitlab Makefile provider-bundle tests README.md client go.mod go.sum providers.sha256sum Dockerfile.templates
ifeq ($(EXCLUDE_CLUSTER_CLASS_FILES), true)
FILES_TO_IGNORE+=yttcc yttcb cconly cc.yaml
endif

FILES_TO_IGNORE_REGEX=$(subst $(space) ,\|,$(FILES_TO_IGNORE))
```

And then update the command.

---

Thanks, definitely easier to read as well. Updated, along with the callsite